dsl/Private.cc

```diff
@@ -28,14 +28,14 @@ vector<unique_ptr<ast::Expression>> Private::replaceDSL(core::MutableContext ctx
             e.setHeader("Use `{}` to define private class methods", "private_class_method");
             auto beginPos = send->loc.beginPos();
             auto replacementLoc = core::Loc{send->loc.file(), beginPos, beginPos + 7};
-            e.replaceWith(replacementLoc, "private_class_method");
+            e.replaceWith("Replace with `private_class_method`", replacementLoc, "private_class_method");
```

Comment:
Are these replacing existing text or adding new text? Should it be "Define with ..."?

Reply:
Nevermind, I saw the test case.